### DISCUSSION TITLE
Upload failed suddenly

### DISCUSSION THREAD

**Discussion by Maurilio97-P** [gavvvr/obsidian-imgur-plugin#89] [Getting help]
In the middle of my note-taking session, the uploads stopped working suddenly:
![image](https://github.com/user-attachments/assets/7fe21546-ae1f-4d9b-8af4-65278c7eb112)
![image](https://github.com/user-attachments/assets/fb1437c7-51a3-489f-acae-2e14106fc4c1)

I have no idea what happened, it was working fine this morning. 
I already had the "Client ID" from the last update, which is why I was able to work with the plugin before. 

What could be the cause of this new issue? 🤔

**Comment by Maurilio97-P** [comment]
UPDATE: I created a second imgur account and now seems to be working. Does imgur have a image limit now? 🤔

**Reply by gavvvr** [reply]
Hi @Maurilio97-P 👋🏻
Rate limits always existed, as I remember. See "rate limits" section at https://apidocs.imgur.com/

I guess you could just simply create another ClientID within the same original account. Based on my experience, I think, rate limits get applied to specific ClientID, not the whole account.
